src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -466,7 +466,7 @@ private void CreateGenericContentFromTemplate(Guid releaseId, Release newRelease
                     await _releaseFilesService
                         .DeleteDataFilesAsync(releaseId, fileName)
                         .OnSuccess(async () => await RemoveFileImportEntryIfOrphaned(deletePlan));
-                    return true;
+                    return Unit.Instance;
```

Comment:
I assume this can just return from the await assuming RemoveFileImportEntryIfOrphaned is changed?

Reply:
Yes it can, thanks. I've made this change.